README.md

```diff
@@ -1,3 +1,5 @@
+# Implementing in another Repo
```

Comment:
Github actions are always added in another repository so I think we can simply change the section title to "Usage"
```suggestion
# Usage
```